pom.xml

```diff
@@ -165,10 +165,10 @@
 	</dependencyManagement>
 
 	<properties>
-		<spring.cloud.commons>1.0.1.BUILD-SNAPSHOT</spring.cloud.commons>
-		<spring.cloud.bus>1.0.1.BUILD-SNAPSHOT</spring.cloud.bus>
-		<spring.cloud.config.client>1.0.1.BUILD-SNAPSHOT</spring.cloud.config.client>
-		<spring.cloud.netflix.core>1.0.1.BUILD-SNAPSHOT</spring.cloud.netflix.core>
+		<spring.cloud.commons>1.0.1.RELEASE</spring.cloud.commons>
```

Comment:
If you rebase with master `.settings.xml` is there now and you can change these back to snapshots